Data-Structure-Stacks.md

```diff
@@ -0,0 +1,60 @@
+# Data Structure Stacks
+
+`Stack` is a linear data structure which follows a particular order in which the operations are performed. The order may be 
+LIFO(Last In First Out) or FILO(First In Last Out).
+
+The basic three operations which can be performed on a `Stack`
+
+1. `Push`: Adds an item in the `Stack`. If the stack is full, then it is said to be an Overflow condition. 
+2. `Pop`: Removes an item from the `Stack`. The items are popped in the reversed order in which they are pushed. If the `Stack` is empty, 
+           then it is said to be an Underflow condition. 
+3. `Peek`: Gets the topmost item of the `Stack`.
```

Comment:
Emphasize that it does not remove the item. Btw, is there some way to get the length or height if the stack? If so, maybe you should mention that as an operation too. Even if it's an auxiliary operation, and not a basic operation specific to Stack.